tests/context/test_context.py

```diff
@@ -35,17 +35,6 @@
 from elasticapm.context.threadlocal import ThreadLocalContext
 
 
-def test_execution_context_backing():
```

Comment:
do we still have to remove this test after you moved opentelemetry to its own matrix entry?